src/MVector.jl

```diff
@@ -18,6 +18,8 @@ end
 let dimension_names = QuoteNode.([:x, :y, :z, :w])
     body = :(getfield(v, name))
     for (i,dim_name) in enumerate(dimension_names)
+        @eval @inline Base.propertynames(v::Union{SVector{$i},MVector{$i}}) = ($(first(dimension_names, i)...),)
```

Comment:
`propertynames` has two methods, I think both have to be added:

```suggestion
        @eval @inline Base.propertynames(v::Union{SVector{$i},MVector{$i}}, ::Bool = false) = ($(first(dimension_names, i)...),)
```

Reply:
https://docs.julialang.org/en/v1/manual/interfaces/#man-instance-properties